src/pages/en/core-concepts/astro-pages.md

```diff
@@ -5,17 +5,30 @@ description: An introduction to Astro pages
 i18nReady: true
 ---
 
-**Pages** are a special type of [Astro component](/en/core-concepts/astro-components/) that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
+**Pages** are special files that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
 
 ### File-based routing
 
-Astro leverages a routing strategy called **file-based routing.** Every `.astro` file in your `src/pages` directory becomes a page or an endpoint on your site based on its file path.
+Astro leverages a routing strategy called **file-based routing.**. Each page file in your `src/pages` directory becomes an endpoint on your site based on its file path.
 
 Write standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) in your component template to link between pages.
 
-
 📚 Read more about [Routing in Astro](/en/core-concepts/routing/).
 
+### Supported page files 
+
+Astro supports the following file types in the `src/pages` directory:
+- [`.astro`](#astro-pages)
+- [`.md`](#markdown-pages)
+- `.mdx` (with the [MDX Integration installed](/en/guides/markdown-content/#mdx-features))
+- [`.js`/`.ts`](/en/core-concepts/astro-pages/#non-html-pages)
+- `.html` (following the same restrictions as [HTML Components](/en/core-concepts/astro-components/#html-components))
```

Comment:
```suggestion
- [`.html`](/en/core-concepts/astro-components/#html-components)
```
I don't think anything from the HTML components section describes anything that would limit it being a page? Yeah, the HTML component section says it's not an Astro component, but... that's kind of what this entire list is about: other things that are not astro files that are pages. I'd be tempted just to remove it. And NOW I'm going to revist that section and add a quick nod to being a full page.